week-4/A-objects-intro/exercise-part-1.js

```diff
@@ -6,4 +6,23 @@ can describe with a JavaScript object
 Assign each of them to a separate variable
 
 */
+var myCar = {
+  colour: "black",
+  make: "kia",
+  model: "ceed",
+  age: 10,
+  isBanger: true,
+  name: "alice"
+};
+var myCat = {
+  colour: "grey",
+  type: "tabby",
+  age: 3,
+  cuddly: true
+};
 
+var mySibblings = {

```

Comment:
Typically, when you have a number of siblings with their various properties, you'd use an array of objects. That would look like this:
```
var mySiblings: [
  {
    name: "Toyin",
    gender: "F",
    location: "USA",
   },
   ...
];
```